ReactiveCocoa.xcodeproj/project.pbxproj

```diff
@@ -1294,7 +1579,6 @@
 					"$(inherited)",
 				);
 				INFOPLIST_FILE = ReactiveCocoaTests/Info.plist;
-				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
```

Comment:
In Xcode 8, you can view the _Customized_ build settings with _Levels_. I'd toggle that and delete anything that's redundant.